lnd.go

```diff
@@ -827,6 +828,48 @@ func getTLSConfig(cfg *Config) (*tls.Config, *credentials.TransportCredentials,
 	}
 
 	tlsCfg := cert.TLSConfFromCert(certData)
+
+	// If Let's Encrypt is enabled, instantiate autocert to request/renew
+	// the certificates.
+	if cfg.LetsEncryptDomain != "" {
+		ltndLog.Infof("Using Let's Encrypt certificate for domain %v",
+			cfg.LetsEncryptDomain)
+
+		manager := autocert.Manager{
+			Cache:      autocert.DirCache(cfg.LetsEncryptDir),
+			Prompt:     autocert.AcceptTOS,
+			HostPolicy: autocert.HostWhitelist(cfg.LetsEncryptDomain),
```

Comment:
The issue with the `autocert` manager is that the certificate is only really requested/generated once the first outside request for one of the white listed domains comes in. This means the user might expect the certificate to be created on startup as before and nothing will happen. Not sure if we should try to force a request here or if we should at least document the behavior.

Reply:
But isn't this completely transparent to the user? Just as if the cert would be expired and a new one requested invisibly?

Reply:
Yes, I guess it would be. But I know we ran into the issue with `aperture` where we thought something didn't work because the certificate wasn't created (automatically). So we should maybe at least mention it somewhere?

Reply:
Added to description of the `letsencryptdomain` flag.